realm/realm-library/src/main/java/io/realm/internal/OsRealmConfig.java

```diff
@@ -218,6 +218,8 @@ private OsRealmConfig(final RealmConfiguration config,
         String urlPrefix = (String)(syncConfigurationOptions[8]);
         String customAuthorizationHeaderName = (String)(syncConfigurationOptions[9]);
         Byte clientResyncMode = (Byte) syncConfigurationOptions[11];
+        String partionKeyValue = (String) syncConfigurationOptions[12];
```

Comment:
Is there a typo in the variable name?

```suggestion
        String partitionKeyValue = (String) syncConfigurationOptions[12];
```